bigbluebutton-html5/imports/ui/components/breakout-room/invitation/component.jsx

```diff
@@ -67,20 +68,17 @@ class BreakoutRoomInvitation extends Component {
     const hasBreakouts = breakouts.length > 0;
 
     if (hasBreakouts && !breakoutUserIsIn && BreakoutService.checkInviteModerators()) {
-      // Have to check for freeJoin breakouts first because currentBreakoutUser will
+      // Have to check for freeJoin breakouts first because currentBreakoutUrl will
       // populate after a room has been joined
-      const breakoutRoom = getBreakoutByUser(currentBreakoutUser);
-      const freeJoinBreakout = breakouts.find(breakout => breakout.freeJoin);
+      const freeJoinBreakout = breakouts.find((breakout) => breakout.freeJoin);
       if (freeJoinBreakout) {
         if (!didSendBreakoutInvite) {
           this.inviteUserToBreakout(breakoutRoom || freeJoinBreakout);
           this.setState({ didSendBreakoutInvite: true });
         }
-      } else if (currentBreakoutUser) {
-        const currentInsertedTime = currentBreakoutUser.insertedTime;
-        const oldCurrentUser = oldProps.currentBreakoutUser || {};
-        const oldInsertedTime = oldCurrentUser.insertedTime;
-        if (currentInsertedTime !== oldInsertedTime) {
```

Comment:
By keeping insertedTime, the user can receive additional invitations to the same or to a different room